### ISSUE TITLE
Won't play next song in queue

### ISSUE THREAD

**Issue by setoret** [Kwoth/NadekoBot#131]
If I queue multiple songs, for example 2, after the first song is finished playing the second will never start. The bot will not say "finished [song]" in the text channel either. Sometimes it will not even play the entire song through, and will end partway into the track without starting the next in queue. After this happens the bot will not respond to !m s or !m n and the only way to get it to play again is to restart the bot. The exe shows up to "Awiting buffer task" and stays there. Screenshot: http://prntscr.com/aiey44

Am I doing anything wrong? I've done several fresh installs and have reverted to previous versions multiple times to no avail. I've triple checked that I have everything installed correctly.

**Comment by Kwoth** [comment]
First time i hear of this. That means ffmpeg process is hanging probaby ,or something else is terribly wrong. Are you sure you have the correct version of ffmpeg for your OS (32/64 bit(?

If you do, i have no idea what is causing this. I will investigate later today, but don't except anything ;[

Very weird

PS Are you hosting on windows or linux, 32 or 64 bit, and what cpu do you have?

**Comment by setoret** [comment]
I am on windows, 64-bit, my cpu is an i5-4690k. I'm certain I installed the 64-bit version of ffmpeg but I'll try reinstalling that now.

**Comment by setoret** [comment]
I just reinstalled ffmpeg, the problem still remains. Honestly not sure what else to try at this point :T

**Comment by samdivaio** [comment]
Assuming you have good internet connection without any blocking, and seeing it playes one song then stoping, have you tried using it on other servers, as a troubleshooter you can try making few discord servers with all different location, starting from Amsterdam and see if that continues to happen there aswell. I have found few people with same issue, with no songs playing after playing one. And they all had issues with discord server location, try "!m d" if you want to force your bot to change server location.

**Comment by setoret** [comment]
Just tried changing the location of the server to every other region and the problem remained across all of them unfortunately.

**Comment by Kwoth** [comment]
Try restarting your pc, helped some people before

**Comment by setoret** [comment]
I've restarted my PC multiple times, sadly made no difference.

**Comment by Kwoth** [comment]
@setoret are you sure you are on latest version of nadeko? You should use nadeko updater to update and copy over your data folder and credentials.json (if you are already on 0.9 releases)

**Comment by setoret** [comment]
Yes, I originally used Nadeko updater to get the latest version. After the problem arose I've done multiple fresh installs both manually and using the updater. I've even downgraded to previous versions hoping it would fix the issue but to no avail.

**Comment by setoret** [comment]
On another note the discord link to your server is invalid or expired. Do you mind providing a fresh link so I can join the server and invite HaNadeko to my server while I get my bot settled?

**Comment by Kwoth** [comment]
which link? http://discordapp.com/invite/0ehQwTK2RBjAxzEY

**Comment by setoret** [comment]
Oh maybe it's just me, when I use that link it says its invalid or expired. http://prntscr.com/ais44q

**Comment by setoret** [comment]
Nevermind, I figured out the link thing. I made a stupid mistake. I'm still having problems with the bot I'm hosting.

**Comment by Kwoth** [comment]
Was your bot banned and you can't join because of that (it bans ips too)? Tell me your and your bot's name and ill unban so we can help you

**Comment by setoret** [comment]
My name is Setoret | Pieri. Discord ID is #4567.
My bot's name is SetoBot. Discord ID is #0986.

**Comment by Kwoth** [comment]
yep setobot was banned, try joining again

**Comment by koellemichael** [comment]
same for me. here is a screenshot from the error http://imgur.com/HNDwNmo

**Comment by Kwoth** [comment]
@koellemichael thats another error, try restarting the bot and changing server location
then queueing again. Afaik, that happens if you didn't give bot permissions.

**Comment by koellemichael** [comment]
@Kwoth thanks! i had to unblock the files. now it works. keep up the good work. awesome bot

**Comment by setoret** [comment]
@Kwoth Hey, I'm still having this issue and was wondering if you've found anything that might be causing it yet. It seems that I'm the only one with this problem so it's probably on my end but I can't for the life of me figure it out.

**Comment by Kwoth** [comment]
Nope man, you are the only person with this issue, there is no reason that should be happening, sorry. You tried everything i could think of. The only other thing i would suggest is signing up for a VPS (like digitalocean) and setting it up there. its 3-5$ per month.

**Comment by Kwoth** [comment]
Are you building from source? I think i may have a hacky fix for you

**Comment by wytesong** [comment]
Originally had the same bug, but then after a while
http://i.imgur.com/2hhqqEw.png

**Comment by Kwoth** [comment]
Did you guys resolve this issue?

**Comment by ThreeOhSix** [comment]
I'm getting the same thing.  I'll give the run down on what i've found.  Setup: running on a digitalocean vps.  Followed the youtube video from the announcements channel to install everything.  Everything was working fine for about 12 hours.  Then this morning was just in channel listening to a radio station via the bot.  I got a discord pm from a friend and the bot's audio started cracking very slightly.  I moved the bot to a different channel, still cracking.  Logged into vps and killed the bot then rebooted.  Now if I que a long youtube video (hour long or so) it will say playing, but in the ssh I get this http://i.imgur.com/BI3lB6x.png .  If i que a couple shorter songs (3-6min) and run !m n it will start playing the first song, but cut it off halfway through.  It will then jump to the next one and play it like normal.

tldr: was working fine last night, issues started the second i received a pm in discord.

**Comment by wytesong** [comment]
Restarted everything, so, Bot's working for me @Kwoth

**Comment by Kwoth** [comment]
Update to the latest release and convert your bot's account to official bot account. The problem should go away.

Tell me if that fixes it.

Here is the guide on how to get the bot account https://github.com/DoNotSpamPls/repository/wiki/How-to-convert-your-bot-account-in-the-API
